src/components/Modal/TorrentDetail/Peers/Peers.svelte

```diff
@@ -49,8 +49,9 @@
             {#if $ipAddress[peer.address]}
               <img
                 class="flag"
-                src="images/flags/{$ipAddress[peer.address].toLowerCase()}.png"
-                alt="{$ipAddress[peer.address]}"
+                src="images/flags/{$ipAddress[peer.address].country.toLowerCase()}.png"
+                alt="{$ipAddress[peer.address].country}"
+                title="{$ipAddress[peer.address].name}"
```

Comment:
Should these actually be the same? Or technially the alt should perhaps say:
```suggestion
                alt="{`Flag of ${$ipAddress[peer.address].name}`}"
                title="{$ipAddress[peer.address].name}"
```

Reply:
What fits into 16x16px?
* Federal Democratic Republic of Ethiopia

or
* ET

🤷

Reply:
ALTS aren't really for fitting into the UI, they are technically there for screen readers.

Reply:
Then consider adding height/width as well:

![alt](https://user-images.githubusercontent.com/5265345/220631632-f14e24bd-c8e3-4e09-aec2-1e6c7ba5ec73.png)

Tho this will most likely mess with the layout nonetheless hence use alpha2 for ALT and ARIA for screen reader

Reply:
Hmm... Fair point!